upup/pkg/fi/cloudup/openstacktasks/lb.go

```diff
@@ -18,9 +18,10 @@ package openstacktasks
 
 import (
 	"fmt"
-	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
 	"time"
 
+	"github.com/gophercloud/gophercloud/openstack/networking/v2/ports"
+
```

Comment:
We probably shouldn't do this in this PR (because it's not thematically related), but if you want to please delete line 24 so that it joins up with the other gophercloud imports.

Reply:
Done, my installed goimports was messing this up for some reason, I removed it from the patch.